From 9ff2ef2f03fc9424d9ae381e2ac89b8e610fc324 Mon Sep 17 00:00:00 2001
From: Philippe Normand <philn@igalia.com>
Date: Fri, 9 Sep 2022 16:32:10 +0100
Subject: [PATCH] pipewire-base: Disable AEC module

The webrtc-echo-canceller hasn't been ported to webrtc-audio-processing-1 yet.
---
 elements/components/pipewire-base.bst | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/elements/components/pipewire-base.bst b/elements/components/pipewire-base.bst
index 78ebf81c7..7bcb5ee73 100644
--- a/elements/components/pipewire-base.bst
+++ b/elements/components/pipewire-base.bst
@@ -29,6 +29,7 @@ build-depends:
 variables:
   meson-local: >-
     -Daudiotestsrc=disabled
+    -Decho-cancel-webrtc=disabled
     -Djack=disabled
     -Droc=disabled
     -Dvideotestsrc=disabled
@@ -60,7 +61,6 @@ public:
       - '%{libdir}/spa-0.2/v4l2'
       - '%{libdir}/spa-0.2/v4l2/**'
       - '%{libdir}/pipewire-0.3/libpipewire-module-access.so'
-      - '%{libdir}/pipewire-0.3/libpipewire-module-echo-cancel.so'
       - '%{libdir}/pipewire-0.3/libpipewire-module-filter-chain.so'
       - '%{libdir}/pipewire-0.3/libpipewire-module-link-factory.so'
       - '%{libdir}/pipewire-0.3/libpipewire-module-portal.so'
@@ -92,7 +92,6 @@ public:
       - '%{libdir}/spa-0.2/v4l2'
       - '%{libdir}/spa-0.2/v4l2/**'
       - '%{libdir}/pipewire-0.3/libpipewire-module-access.so.debug'
-      - '%{libdir}/pipewire-0.3/libpipewire-module-echo-cancel.so.debug'
       - '%{libdir}/pipewire-0.3/libpipewire-module-filter-chain.so.debug'
       - '%{libdir}/pipewire-0.3/libpipewire-module-link-factory.so.debug'
       - '%{libdir}/pipewire-0.3/libpipewire-module-portal.so.debug'
-- 
2.37.1

